cmd/ci-operator/main.go

```diff
@@ -1588,6 +1596,24 @@ func getHashFromBytes(b []byte) string {
 	return oneWayNameEncoding.EncodeToString(hash.Sum(nil)[:5])
 }
 
+func getPushSecret() (*coreapi.Secret, error) {
```

Comment:
Can this be DRYed out with `getPullSecretFromFile` and the `Secret` creation code?

Reply:
Done.